### ISSUE TITLE
Documentation: Document installation of mapnik (esp. for Windows)

### ISSUE THREAD

**Issue by jmunowitch** [archesproject/arches#1819]
### Bug Report
When installing arches on Windows, the Tile Server does not come up.  The culprit seems to be Mapnik not installing.

#### Expected Behavior
Tile server should work

#### Current Behavior
![Error importing Mapnik](https://cloud.githubusercontent.com/assets/1542229/23965416/21a563a0-0975-11e7-9fb8-c5b5310ba7cd.png)

#### Steps to Reproduce (for bugs)
1. Install on windows
2. import a layer into tileserver such as `python manage.py packages -o add_tileserver_layer -m arches/tileserver/hillshade.xml -n hillshade`
3. attempted to access the layer, such as `curl -I localhost:8000/tileserver/hillshade/11/324/790.png`

#### Your Arches Information
* Version used: Latest Master
* Browser Name and version: Not Applicable
* Operating System and version (desktop or mobile): Windows 10 64 bit
* Python Version: 64 bit

**Comment by jmunowitch** [comment]
This may just be an install process issue. if that is the case, we just need to add a few lines into the documentation to fix this.

**Comment by robgaston** [comment]
Yes - we just need to properly document the mapnik install steps.

**Comment by jmunowitch** [comment]
@mradamcox can speak to this more. When we spoke this morning, he was reproducing and confirming this was the case.

**Comment by robgaston** [comment]
related to #1696

**Comment by mradamcox** [comment]
Summary to this point:
We have been working with mapnik 3.0.x. We have found that installing mapnik through node works on my laptop, as well as using one of the binaries that Jeff found. However, neither of us have been able to install the python bindings.

`pip install mapnik` seems to work, but `import mapnik` results in the error @jmunowitch posted above (on both 32 and 64-bit python).

All documentation I've found states that 32-bit Python is needed anyway (I didn't have luck with that either) [here](https://github.com/mapnik/mapnik/issues/3127) and [here](https://gis.stackexchange.com/questions/212151/mapnik-on-64-bit-windows) for example. We know that on 64-bit Windows, 64-bit python **must** be used in order to make the app run. To hopefully get to the bottom of this, I have created [this issue](https://github.com/mapnik/python-mapnik/issues/145) in the python-mapnik repo. It definitely seems possible that python bindings for windows with 64-bit python is not an option.
